### PR TITLE
feat(gui): rework manual upgrade to use polling requests

### PR DESCRIPTION
**Pull request by william-gr**
Apparently some recent combination of browsers+OS can make long running
HTTP POST responses be entirely missed by the browser.

Ticket:	#29040

### CODE REVIEW

File: gui/system/views.py [hunk 1]
```diff
@@ -973,6 +973,58 @@ def file_browser(request, path='/'):
...
 
 
+def manualupdate_running(request):
+    uuid = request.GET.get('uuid')
+    if not uuid:
+        return HttpResponse(uuid, status=202)
+
+    _n = notifier()
+    if not _n.is_freenas() and _n.failover_licensed():
+        if uuid != "0":
```
Comment (skarekrow) [inline gui/system/views.py]: `uuid` is an int above, here you're matching a string?

File: gui/system/views.py [hunk 2]
```diff
@@ -973,6 +973,58 @@ def file_browser(request, path='/'):
...
+                    if job['state'] == 'SUCCESS':
+                        try:
+                            c.call('failover.call_remote', 'system.reboot', [{'delay': 2}])
+                        except Exception:
+                            log.debug('Failed to reboot standby', exc_info=True)
+                        return render_to_response('failover/update_standby.html')
+                    elif job['state'] == 'FAILED':
+                        return JsonResp(request, message=job['error'], error=True)
+        else:
+            # XXX: very ugly hack to get the legacy manuak upgrade thread from the form
```
Comment (skarekrow) [inline gui/system/views.py]: s/manuak/manual

File: gui/system/views.py [hunk 3]
```diff
@@ -973,6 +973,58 @@ def file_browser(request, path='/'):
...
+            from freenasUI.system.forms import LEGACY_MANUAL_UPGRADE
+            if LEGACY_MANUAL_UPGRADE:
+                if not LEGACY_MANUAL_UPGRADE.isAlive():
```
Comment (skarekrow) [inline gui/system/views.py]: These two could probably be combined into one statement

File: gui/system/views.py (hunk 1 above)
Comment (william-gr) [reply gui/system/views.py]: uuid is a str here, not GET params are int, not possible.

**Pull request merged**